src/aat/resources/features/F-048/F-048.feature

```diff
@@ -0,0 +1,56 @@
+@F-048
+Feature: Get the pagination metadata for a case data search for Case Worker
+
+  @S-133
+  Scenario: must return 400 if the sort direction in input parameters is not in ASC or DESC
+    Given an appropriate test context as detailed in the test data source
```

Comment:
This standard first line should indeed be moved to Background section as @noronhaa suggested.